decidim-blogs/config/locales/en.yml

```
@@ -40,6 +40,11 @@ en:
           update:
             invalid: There was a problem saving the post.
             success: Post successfully saved
+      admin_log:
+        post:
+          create: "%{user_name} created the %{resource_name} blog post in %{space_name}"
+          delete: "%{user_name} removed the %{resource_name} blog post from %{space_name}"
```

I see that in other places we use `deleted` instead of `removed`, so I prefer that to be consistent with other admin log actions. The only places that I found that use `remove` is to say `removed from X`.  Can you change it please? Thanks.

```suggestion
          delete: "%{user_name} deleted the %{resource_name} blog post from %{space_name}"
```